aten/src/ATen/native/nested/NestedTensorMath.cpp

```diff
@@ -864,9 +864,12 @@ Tensor matmul_nested(const Tensor& self, const Tensor& mat2) {
   else if (!self.is_nested() && mat2.is_nested()) {
     AT_ERROR("Expected both to be nested, but got a non-nested self and nested other");
   }
+  // to_padded_tensor only supports contiguous inputs
```

Comment:
Just calling out that these additional contig calls are necessary, we do use matmul with noncontig inputs in the at::sdp op  as well as nested_matmul backward?